src/Avalonia.Vulkan/IVulkanContextExternalObjectsFeature.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using Avalonia.Metadata;
+using Avalonia.Platform;
+using Avalonia.Rendering.Composition;
+
+namespace Avalonia.Vulkan;
+
+[Unstable]
```

Comment:
We should start marking Unstable attribute with a specific reason why. Here and in other places.